ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt

```diff
@@ -344,6 +344,10 @@ r, ".*ERR swss\d*#orchagent:.*handlePortStatusChangeNotification: Failed to get
 # Ignore SAI_NEXT_HOP_GROUP_ATTR_TYPE unsupported
 r, ".* ERR swss#orchagent:.*queryAttributeEnumValuesCapability:.*returned value \d+ is not allowed on SAI_NEXT_HOP_GROUP_ATTR_TYPE"
 
+# Ignore SRV6 error
+r, ".* ERR syncd#syncd: \[none\] SAI_API_SRV6:_brcm_sai_srv6_config_get:\d+ SRV6 unsupported or not enabled on this device"
```

Comment:
Can we remove the [none] here to prevent the log from appearing multiple times and being aggregated?